examples/with-apollo-auth/lib/init-apollo.js

```diff
@@ -0,0 +1,48 @@
+import { ApolloClient, createNetworkInterface } from 'react-apollo'
+import fetch from 'isomorphic-fetch'
+
+let apolloClient = null
+
+// Polyfill fetch() on the server (used by apollo-client)
+if (!process.browser) {
+  global.fetch = fetch
+}
+
+function create (initialState, { getToken }) {
+
+  const networkInterface = createNetworkInterface({
+    uri: 'https://api.graph.cool/simple/v1/cj3h80ffbllm20162alevpcby',
```

Comment:
I'd recommend here that this doesn't use a specific project ID and uses an ENV variable or some instructions inside the README pointing people to use your schema with their CLI.

```
graphcool init --schema schema.graphql
```